File: Logic Building/Basic Problems/Multiplication Table/multy_table.c
#include <stdio.h>


int multy_table(int n);  // prototype

// defination
int multy_table(int n){

    for (int i = 0; i < 11; i++)
    {
        printf("The multyplication using function: ");
        printf("%d x %d = %d\n",n,i,n*i);
    
    }
}

int main(){
    // Procigral Programming
    int num;
    scanf("%d",&num);

    for (int i = 0; i < 11; i++)
    {
        printf("%d x %d = %d\n",num,i,num*i);
        
    }

    
    // functional Approach
    multy_table(num);



    
    return 0;
}

File: Logic Building/Basic Problems/Multiplication Table/multy_table.py
number = int(input("Enter Number: "))

# Procigral Approach
# for i in range(1,11):
#     print(f"{number} x {i} = {number*i}")

# Functional Approach

def multy_table(n):
    for i in range(1,11):
        print(f"{n} x {i} = {n*i}")
    return n*i

print(multy_table(number))